src/components/Login/Login.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import GoogleLogin from "react-google-login";
+import { FcGoogle } from 'react-icons/fc'
+import styles from './Login.module.css'
+/**
+ * The Login button component can be used to authenticate the user using a Google oAuth. 
+ * The action of what happens if the user is succesfully authenticates (onSuccess) and 
+ * if the authentication fails (onFailure) are passed as props which have the following structure:
+ * 
+ * Example of onSuucess prop that would print the profileObj and tokenObj on the console
+ * const onSuccess = (res) => {
+ *      console.log(res.profileObj)
+ *      console.log(res.tokenObj)
+ * }
+ * 
+ * The "react-google-login" package is used that provides simple mechanisms to add the Google login button.
+ * For more impormation visit: "https://www.npmjs.com/package/react-google-login"
+ * 
+ * The clientId was obtained from Google developer console, that is used to identify the application for authentication details..
+ * The isSignedIn prop is used to ensure the user stays logged in.
+ * The render prop used is used to set up a customized Google login button.
+ * 
+ * The "react-icons" package was used to use the google icon "FcGoogle" on the customized sign in button.
+ * 
+ * After clicking on the button, a window that allows the user to sign in with Google opens up.
+ * After signing up the user will be redirected to the same page they login in from (i.e. http://localhost:3000/), 
+ * which can be changed from Google Developer Console later. 
+ * 
+ */
+
+const clientId = "158309441002-q8q49tjicngt1tp6p9t7ecvdrn9ar78j.apps.googleusercontent.com";
+
+export default function Login({onSuccess, onFailure}){
```

Comment:
Can we maybe rename this component to `LoginButton` because we will have another component soon which will also be called `Page`, so it may become confusing. Also can we move this to `src/pages/Login/LoginButton` because its not reusable and it will only really be used on the `Login` page component.